exe/deprecations

```diff
@@ -76,6 +83,10 @@ option_parser = OptionParser.new do |opts|
     options[:verbose] = true
   end
 
+  opts.on("--path PATH_OR_FILE_GLOB", "Path to deprecation shitlist or glob for multiple files") do |path|
+    options[:path] = path
+  end
```

Comment:
I think we should put the use of this option in Readme as well.

Reply:
+1